src/App.tsx

```diff
@@ -1,10 +1,49 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { Status } from './types/Status';
 
-const USER_ID = 0;
+const USER_ID = 11092;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState(false);
+  const [selectItem, setSelectItem] = useState(Status.ALL);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch((error) => {
+        setErrorMessage(true);
+        throw error;
+      });
+  }, []);
+
+  const isActiveTodos = todos.filter(todo => !todo.completed);
+
+  const preparedTodos = todos
+    ?.filter(todo => {
```

Comment:
Seems that todos will always exist
```suggestion
  const preparedTodos = todos.filter(todo => {
```